Controller/HelperController.php

```diff
@@ -53,12 +54,18 @@ class HelperController
     protected $validator;
 
     /**
+     * @var TwigRenderer
+     */
+    protected $twigRenderer;
+
+    /**
      * @param \Twig_Environment  $twig
      * @param Pool               $pool
      * @param AdminHelper        $helper
      * @param ValidatorInterface $validator
+     * @param TwigRenderer       $twigRenderer
      */
-    public function __construct(\Twig_Environment $twig, Pool $pool, AdminHelper $helper, $validator)
+    public function __construct(\Twig_Environment $twig, Pool $pool, AdminHelper $helper, $validator, TwigRenderer $twigRenderer)
```

Comment:
And use `TwigRendererInterface` as type hinting, it will be easier to make the build pass